opencsp/app/sofast/lib/BlobIndex.py

```diff
@@ -283,19 +271,16 @@ def _find_3x3_center_block(self) -> None:
         self._assign(idx_g, x, y)
 
     def _extend_data(self, direction: Literal['x', 'y'], step: Literal[1, -1]) -> None:
-        """Extends found blob rows/collumns in given direction
-
-        Steps in the given axis, a or b
-
-        Parameters
-        ----------
-        direction : Literal['x', 'y']
-            Axis to search
-        step : Literal[1, -1]
-            Direction to search
-            -  1 = right/down
-            - -1 = left/up
-        """
+        # Extends found blob rows/collumns in given direction
+        # Steps in the given axis, a or b
+        # Parameters
+        # ----------
+        # direction : Literal['x', 'y']
+        #     Axis to search
+        # step : Literal[1, -1]
```

Comment:
consider a enum for step:
```
class Step(Enum):
    RIGHT_OR_DOWN = 1
    LEFT_OR_UP = -1
```

Reply:
I have not used Enum before; I'll give it a try.